src/components/ProductDetail.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+function ProductDetail(props){
+
+    var starMarkup = '';
+    for (var i = 0, j = props.product.rating; i < j; i++) {
+        starMarkup += '★';
```

Comment:
Make note of what markup actually made the star images show up. jsx allows you to manipulate html elements as javascript also. so you could have done something like
myarray.push(<span/>)